### ISSUE TITLE
KOReader not working on PocketBook 623

### ISSUE THREAD

**Issue by KitezhGrad** [koreader/koreader#3539]
* KOReader version : koreader-pocketbook-arm-obreey-linux-gnueabi-v2015.11-1370-g5ee6c01
* Device: PocketBook 623

**Issue**

Unable to launch KOReader on PocketBook 623.

**Steps to reproduce**

1) Follow the wiki to install Koreader (basically copying /system and /applications).
2) Books open with AdobeViewer (the default reader program on PB 623)
Also, KOReader refuses to open from the application menu.

**crash.log** 
[crash.log](https://github.com/koreader/koreader/files/1542777/crash.log)

**Comment by AlanSP1** [comment]
This looks like duplicate of this issue: https://github.com/koreader/koreader/issues/3537

But, there it works if koreader is started and book opened with koreader's file manager.

**Comment by mk-f** [comment]
Nope, PB 623 is actually not detected at the moment (it worked before as we had virtually no detection at all and all pocketbooks used the same preferences (dpi etc)). I'll see to it this weekend.

**Comment by mk-f** [comment]
@KitezhGrad Can you try with [this](https://github.com/mk-f/koreader/blob/f7a84c90c5b0c3d09ebffe3f15365fee24347b9c/frontend/device/pocketbook/device.lua) device.lua? Replace the one under applications/koreader/frontend/device/pocketbook/device.lua on you reader.

**Comment by KitezhGrad** [comment]
@mk-f I've replaced the file, and here are the results:

1. I've been able to launch KOReader from the application menu and open some books from within the application.
2. However, KOReader still doesn't appear in the "Open with" window along with AdobeViewer and fbreader (the pre-installed reader programs on PB 623) when I view my books in the default PocketBook Library. 
Is there a way to make KOReader my default reader program, so I don't have to go to page 2 of the Applications menu every time I want to open a book?
